resources/styles/components/task-plan/homework/exercise-summary.less

```diff
@@ -1,5 +1,5 @@
 .homework-plan .exercise-controls-bar {
-
+  height: 90px;
   flex-wrap: wrap;
```

Comment:
90px?  That seems (and looks) really big to me.  It's also causing some issues with the section titles being barely visible.  If 90 is really desired, then I think we'll also need to add some margin-top on the section so the title is more visible. Right now it looks like (also not a fan of the huge margin on sides):

![screen shot 2016-07-21 at 5 11 47 pm](https://cloud.githubusercontent.com/assets/79566/17040604/3b389ef0-4f66-11e6-99e8-b3657c8602ea.png)